app/javascript/src/components/App.tsx

```
@@ -46,40 +47,59 @@
+            handleOverlayVisibility,
+            selectedTab,
+            setSelectedTab,
+            company,
+            setCompany,
+          }}
+        >
+          <AuthProvider>
+            <BrowserRouter>
+              <Toaster
+                position="top-right"
+                duration={5000}
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)